server/services/utils/conditionsValidator.js

```diff
@@ -0,0 +1,27 @@
+const validator = require('validator');
+module.exports = {validate};
+
+function validate(inputObject, conditionsSelected) {
+
+    const missing = requiredInputMissing(inputObject, conditionsSelected);
+    if (missing.length > 0) {
+        return {
+            validates: false,
+            missing
+        };
+    }
+
+    return {
+        validates: true
```

Comment:
I would pass the `inputObject` back here, gives you a place to do type co-ercions